fix(docs): updated upgrade guide RedHatDisplay text (#3195)
patternfly/patternfly: UPGRADE-GUIDE.md

### Default font

> We've updated the default font to be `RedHatText` instead of `Overpass` [(#2955)](https://github.com/patternfly/patternfly/pull/2955). To use the old font, add the class `pf-m-overpass-font` on an element that wraps your application (for example, `<body>`).

Reviewer: Just mentioning `RedHatDisplay` here with a couple of small edits

```suggestion
We've updated the default font from `Overpass` to `RedHatText` and `RedHatDisplay` [(#2955)](https://github.com/patternfly/patternfly/pull/2955). To continue to use `Overpass`, add the class `pf-m-overpass-font` on an element that wraps your application (for example, `<body>`).
```

> You don’t have to do anything further to use this font. However, with the change from  `Overpass`  to  `RedHatText` , we encourage you to review your application’s typography styles to ensure they are correct.

Reviewer: ```suggestion
You don’t have to do anything further to use this font. However, with the change from  `Overpass`  to  `RedHatText` and `RedHatDisplay`, we encourage you to review your application’s typography styles to ensure they are correct.
```
